CMakeLists.txt

```diff
@@ -4,8 +4,7 @@
 
 cmake_minimum_required(VERSION 2.8)
 
-SET(PROJECTNAME wholeBodyModel)
-PROJECT(${PROJECTNAME})
+PROJECT(wholeBodyModel)
```

Comment:
The `PROJECT_NAME` variable is used in the cmake to name the mex file. If we want to call it `mexWholeBodyModel`, we will have to change also all the matlab files.